static_files/index.html

```diff
@@ -16,37 +16,33 @@
     <!-- Header -->
 
     <header>
-      <div>
-
-        <!-- Logo -->
-
-        <div>
+<!-- Always Visible Header -->
+      <div class="HeaderContainer">
+        <div class="CathoLogo">
           <a href="#"
             ><img
               src="../static_files/imgs/cathoLogo.svg"
               alt="Logotipo da Catho"
           /></a>
         </div>
-        <!-- Main Navigation Menu -->
-        <nav>
+        <nav class="VisibleMenu">
           <ul>
             <li><a href="#">Catho Components</a></li>
             <li><a href="#">Quantum</a></li>
+            <li>&#x2630;</li>
```

Comment:
Styling this item on hover is not working. Should this be the expected behavior for this task? If so, how about creating a new class specifically for this element and styling it as needed? Or instead of styling the "a" tag how do you style the "li" tag with the required color?

Reply:
fixed creating a style for this item list